src/vs/platform/list/browser/listService.ts

```diff
@@ -490,6 +490,8 @@ export class WorkbenchPagedList<T> extends PagedList<T> {
 			this.updateStyles(options.overrideStyles);
 		}
 
+		this.getHTMLElement().classList.toggle('default-styled-list', !options.overrideStyles);
```

Comment:
But if I only set one color in overrideStyles, then do I lose all the default colors?